packages/cozy-konnector-libs/src/libs/saveBills.js

```diff
@@ -56,6 +56,9 @@ module.exports = (entries, fields, options = {}) => {
     delete entry.fileDocument
     return entry
   }
+  entries.map(entry => {
```

Comment:
:warning: This will not work in konnectors where we pass an array of promise (like https://github.com/konnectors/auchandrive/blob/3b9d41d3e2dd0d2804bfcfe442a57bdb5e928b0d/src/index.js#L36)

Put it in `postProcess` instead.